frontend/src/fixtures/riderFixtures.js

```diff
@@ -12,7 +12,48 @@ const riderFixtures = {
         "course": "CMPSC111"    
       }
     ],
+    threeRidesTable:
```

Comment:
Are these riderFixtures or rideFixtures?

They look like rideFixtures, especially since this variable is called `threeRidesTable`

You might think that names aren't important, but they actually are: when the name of something doesn't match what it represents, this causes extra cognitive load for future programmers, and makes code harder to understand.

Could we change the name of this file to rideFixtures?

You can guess what I'm going to ask you to do next.

Reply:
Commit [007145b](https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/pull/32/commits/007145b7af202a2285f77a7ca76d7e8f4b67990f) should fix this